<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="hide_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>293</y>
      <width>471</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelException">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>67</width>
       <height>44</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonRadiant">
        <property name="text">
         <string>Radiant</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDegre">
        <property name="text">
         <string>Degré</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="show_keyboard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>471</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligne de commande:</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>97</width>
       <height>44</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButtonComplexe">
        <property name="text">
         <string>Complexe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonNonComplexe">
        <property name="text">
         <string>Non Complexe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>68</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonEntiere">
        <property name="text">
         <string>Entière</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRationnelle">
        <property name="text">
         <string>Rationnelle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonReelle">
        <property name="text">
         <string>Réelle</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="keyBoard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>491</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>453</width>
       <height>269</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="button9">
            <property name="text">
             <string>9</string>
            </property>
            <property name="shortcut">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="button8">
            <property name="text">
             <string>8</string>
            </property>
            <property name="shortcut">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="button7">
            <property name="text">
             <string>7</string>
            </property>
            <property name="shortcut">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="button6">
            <property name="text">
             <string>6</string>
            </property>
            <property name="shortcut">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="button5">
            <property name="text">
             <string>5</string>
            </property>
            <property name="shortcut">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="button4">
            <property name="text">
             <string>4</string>
            </property>
            <property name="shortcut">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="button3">
            <property name="text">
             <string>3</string>
            </property>
            <property name="shortcut">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="button2">
            <property name="text">
             <string>2</string>
            </property>
            <property name="shortcut">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="button1">
            <property name="text">
             <string>1</string>
            </property>
            <property name="shortcut">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="buttonEntrer">
            <property name="text">
             <string>Entrer</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="buttonAnnuler">
            <property name="text">
             <string>Annuler</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Z</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="buttonRetablir">
            <property name="text">
             <string>Rétablir</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Y</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="buttonDollar">
            <property name="text">
             <string>$</string>
            </property>
            <property name="shortcut">
             <string>$</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="button0">
            <property name="text">
             <string>0</string>
            </property>
            <property name="shortcut">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="buttonVirgule">
            <property name="text">
             <string>,</string>
            </property>
            <property name="shortcut">
             <string>,</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="divButton">
            <property name="text">
             <string>/</string>
            </property>
            <property name="shortcut">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonPow">
            <property name="text">
             <string>Pow</string>
            </property>
            <property name="shortcut">
             <string>P, O, W</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mulButton">
            <property name="text">
             <string>*</string>
            </property>
            <property name="shortcut">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonMod">
            <property name="text">
             <string>Mod</string>
            </property>
            <property name="shortcut">
             <string>M, O, D</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="subButton">
            <property name="text">
             <string>-</string>
            </property>
            <property name="shortcut">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>+</string>
            </property>
            <property name="shortcut">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonSign">
            <property name="text">
             <string>Sign</string>
            </property>
            <property name="shortcut">
             <string>S, I, G, N</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="buttonExp">
            <property name="text">
             <string>'</string>
            </property>
            <property name="shortcut">
             <string>'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonSin">
          <property name="text">
           <string>Sin</string>
          </property>
          <property name="shortcut">
           <string>S, I, N</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonSinh">
          <property name="text">
           <string>Sinh (sh)</string>
          </property>
          <property name="shortcut">
           <string>S, H</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonLn">
          <property name="text">
           <string>Ln</string>
          </property>
          <property name="shortcut">
           <string>L, N</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonSqrt">
          <property name="text">
           <string>Sqrt</string>
          </property>
          <property name="shortcut">
           <string>S, Q, R, T</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonFacto">
          <property name="text">
           <string>!</string>
          </property>
          <property name="shortcut">
           <string>!</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonCos">
          <property name="text">
           <string>Cos</string>
          </property>
          <property name="shortcut">
           <string>C, O, S</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonCosh">
          <property name="text">
           <string>Cosh (ch)</string>
          </property>
          <property name="shortcut">
           <string>C, H</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="buttonLog">
          <property name="text">
           <string>Log</string>
          </property>
          <property name="shortcut">
           <string>L, O, G</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="buttonSqr">
          <property name="text">
           <string>Sqr</string>
          </property>
          <property name="shortcut">
           <string>S, Q, R</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="buttonEval">
          <property name="text">
           <string>EVAL</string>
          </property>
          <property name="shortcut">
           <string>E, V, A, L</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonTan">
          <property name="text">
           <string>Tan</string>
          </property>
          <property name="shortcut">
           <string>T, A, N</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonTanh">
          <property name="text">
           <string>Tanh (th)</string>
          </property>
          <property name="shortcut">
           <string>T, H</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="buttonInv">
          <property name="text">
           <string>Inv</string>
          </property>
          <property name="shortcut">
           <string>I, N, V</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="buttonCube">
          <property name="text">
           <string>Cube</string>
          </property>
          <property name="shortcut">
           <string>C, U, B, E</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="buttonPI">
          <property name="text">
           <string>PI</string>
          </property>
          <property name="shortcut">
           <string>P, I</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="spaceButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Espace</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
